programs/sbf/c/src/invoked/invoked.c

```diff
@@ -45,7 +45,6 @@ extern uint64_t entrypoint(const uint8_t *input) {
     sol_assert(accounts[ARGUMENT_INDEX].data_len == 100);
     sol_assert(accounts[ARGUMENT_INDEX].is_signer);
     sol_assert(accounts[ARGUMENT_INDEX].is_writable);
-    sol_assert(accounts[ARGUMENT_INDEX].rent_epoch == 0);
```

Comment:
Hey @ryoqun & @jackcmay . I want to stop testing the behavior of rent_epoch in these tests. I'm trying to get rid of this field.

Reply:
these changes seem to set the rent epoch to a different value which is still sent to the program.  Why not test the new expected behavior, if nothing less to ensure consistent and expected behavior?